src/components/experiment-tracking/tooltip/tooltip.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import classnames from 'classnames';
+
+import './tooltip.css';
+
+export const tooltipDefaultProps = {
+  content: { label1: '', value1: '', label2: '', value2: '' },
+  direction: 'right',
+  pos: { x: -500, y: -500 },
+  visible: false,
+};
+
+export const MetricsChartsTooltip = ({
+  content = tooltipDefaultProps.content,
+  visible = tooltipDefaultProps.visible,
+  pos = tooltipDefaultProps.pos,
+  direction = tooltipDefaultProps.direction,
```

Comment:
```suggestion
  content = tooltipDefaultProps.content,
  direction = tooltipDefaultProps.direction,
  position = tooltipDefaultProps.position,
  visible = tooltipDefaultProps.visible,
```


I think you can spell out **position** here, and then elsewhere where it's used.